app/src/main/res/layout/tab_tray_item.xml

```diff
@@ -61,12 +62,14 @@
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_alignParentTop="true"
+        android:layout_marginTop="12dp"
+        android:layout_marginStart="12dp"
+        android:layout_marginEnd="8dp"
         android:ellipsize="end"
-        android:lines="1"
-        android:paddingStart="16dp"
-        android:paddingTop="22dp"
+        android:lines="2"
         android:textColor="@color/tab_tray_item_text_normal_theme"
         android:textSize="16sp"
+        android:lineHeight="20sp"
```

Comment:
This is only for sdk >27.  You should use `lineSpacingExtra `

Reply:
Is now fixed